docs/http/traffic-policy/index.mdx

```diff
@@ -23,7 +23,7 @@ import TrafficPolicyWarning from "/shared/warnings/traffic-policy-preview.mdx";
 ## Overview
 
 This module allows you to assign a policy to your endpoints containing a series
-of inbound and outbound rules that can be used to influence and control traffic
+of `on_tcp_connect`, on_http_request`, and `on_http_response` rules that can be used to influence and control traffic
```

Comment:
Missing a backtick before `on_http_request`!

This would be a good place to use the phase terminology. Maybe something like ...

"This module enables you to assign a policy to your endpoints by defining a set of rules for three traffic management phases: `on_tcp_connect`, `on_http_request`, and `on_http_response`. These rules allow you influence and control traffic to and from your upstream service."

Reply:
I like that, done.